shell/renderer/api/electron_api_context_bridge.cc

```diff
@@ -687,17 +687,21 @@ v8::MaybeLocal<v8::Object> CreateProxyForAPI(
           continue;
         }
       }
-      v8::Local<v8::Value> value;
-      if (!api.Get(key, &value))
-        continue;
-
-      auto passed_value = PassValueToOtherContextInner(
-          source_context, source_execution_context, destination_context, value,
-          api.GetHandle(), object_cache, support_dynamic_properties,
-          recursion_depth + 1, error_target);
-      if (passed_value.IsEmpty())
-        return {};
-      proxy.Set(key, passed_value.ToLocalChecked());
+
+      {
+        v8::Context::Scope source_context_scope(source_context);
+        v8::Local<v8::Value> value;
+        if (!api.Get(key, &value))
+          continue;
+
+        auto passed_value = PassValueToOtherContextInner(
+            source_context, source_execution_context, destination_context,
+            value, api.GetHandle(), object_cache, support_dynamic_properties,
+            recursion_depth + 1, error_target);
+        if (passed_value.IsEmpty())
+          return {};
+        proxy.Set(key, passed_value.ToLocalChecked());
```

Comment:
I think _technically_ this `proxy.Set` call should be within a `destination_context_scope`

Reply:
Done in [a152cb4](https://github.com/electron/electron/pull/46253/commits/a152cb4c48c6c2fea09c579500b9fd19e8a5f1a2)